### DISCUSSION TITLE
How do you perform simple text search for models using listModels?

### DISCUSSION THREAD

**Discussion by mikjee** [huggingface/huggingface.js#506] [General]
I am looking at the listModels method in the hubs API, and it seems that the JS version of huggingface package has no way to search using a string. I would like to search, for example, using a string query "llava v1.6". The PY API has a search field which takes a string, but the js API takes an object. 

Am I missing something?

(https://github.com/huggingface/huggingface.js/blob/8ad5507731cd7b2636b6fafb9d4842e6fecd0ef1/packages/hub/src/lib/list-models.ts#L53)

**Comment by coyotte508** [comment]
Opened this PR: https://github.com/huggingface/huggingface.js/pull/507

**Reply by coyotte508** [reply]
Released in version 0.14.2!

```ts
listModels({search: {query: "llava v1.6"}})
```

**Reply by mikjee** [reply]
Thank you a lot! So far I was monkey-patching the fetch object, I appreciate adding this :)
